udata/core/dataset/csv.py

```diff
@@ -1,4 +1,4 @@
-from udata.core.discussions.models import Discussion
+from udata.core.discussions.csv import DiscussionCsvAdapter  # noqa: F401
```

Comment:
Could you add a comment that it's for backward compatibility (maybe with a link to this PR?)